operator/src/main/java/org/bf2/operator/operands/KafkaCluster.java

```diff
@@ -554,6 +562,15 @@ private KafkaClusterTemplate buildKafkaTemplate(ManagedKafka managedKafka, int r
         return templateBuilder.build();
     }
 
+    private List<ContainerEnvVar> buildKafkaContainerEnvVars(ManagedKafka managedKafka) {
+        var kafkaOverride = this.overrideManager.getKafkaOverride(managedKafka.getSpec().getVersions().getStrimzi());
```

Comment:
I tried approaches introducing an env var type parameter to `OperandOverride<E>`) etc but the resulting code `applyEnvironmentTo` in  less obvious that this list conversion.